colossalai/kernel/cuda_native/csrc/attention_infer_kernels/layernorm/layernorm_kernels.cu

```diff
@@ -0,0 +1,62 @@
+/*This code from Vllm : https://github.com/vllm-project/vllm
+ *     with minor changes. */
```

Comment:
We already had layernorm kernels. Is this more efficient?

Reply:
yes, it is more efficient than apex layernorm